src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -291,6 +291,7 @@ export default class CrossDeviceMobileRouter extends Component<
           'metadata',
           'method',
           'side',
+          'variant',
```

Comment:
oh, were we not returning the variant from the cross device client? 😱

Reply:
Nope, my bad, the correct `variant` was extracted from `metadata`, not from the `capture` state. Removed